Fix prepareCustoms key for Single axis.

benlongo opened the pull request:
## Brief Information

This pull request is in the type of:

- [x] bug fixing
- [ ] new feature
- [ ] others

### What does this PR do?

When using a custom series with a single axis, a lookup on `prepareCustoms` fails as it is looking for `single` instead of `singleAxis`.


### Fixed issues

- custom series on single axis now works

### Related test cases or examples to use the new APIs

If this is the right approach to fixing this, I can put together a reproduction and test for it.

pissang commented:
@benlongo Thanks for the contribution and LGTM!. Adding a new test case for this fix will be great! You can append this test case to `test/custom.html`

benlongo commented:
Sorry for the long delay! I've added a test and confirmed that it is broken on master and that this branch fixes it. I additionally fixed a typo I came across but can pull that into a separate PR if you prefer. 

I modified an existing example to reproduce the, which isn't a minimal, but I can make it simpler if required.

pissang commented:
@benlongo It's fine. All looks good! Well done!
